File: tests/integration/deleted_with/python/fails_on_delete.py
# Copyright 2016-2023, Pulumi Corporation.  All rights reserved.

from typing import Optional

import khulnasoft

class FailsOnDelete(khulnasoft.CustomResource):
    def __init__(self,
                 resource_name: str,
                 opts: Optional[khulnasoft.ResourceOptions] = None):
        super().__init__("testprovider:index:FailsOnDelete", resource_name, {}, opts)

File: tests/integration/deleted_with/nodejs/failsOnDelete.ts
// Copyright 2016-2023, Pulumi Corporation.  All rights reserved.

import * as khulnasoft from "@khulnasoft/khulnasoft";

export class FailsOnDelete extends khulnasoft.CustomResource {
    constructor(name: string, opts?: khulnasoft.CustomResourceOptions) {
        super("testprovider:index:FailsOnDelete", name, {}, opts);
    }
}
